tests_common/pytest_plugin.py

```diff
@@ -40,7 +40,7 @@
     from airflow.models.dag import DAG, ScheduleArg
     from airflow.models.dagrun import DagRun, DagRunType
     from airflow.models.taskinstance import TaskInstance
-    from airflow.operators.empty import EmptyOperator
+    from airflow.providers.standard.operators.empty import EmptyOperator
```

Comment:
why import it here if it is imported lazily in the file inside functions?

Reply:
Not particularly done on purpose. It was a migration effort, so the old occurences => new occurences were made.
Anyway, unified it

Reply:
Thats because it is in a type checking block used for mypy etc. But we import it normally in functions that need it for sake of usage